src/gui/qgsfilecontentsourcelineedit.cpp

```diff
@@ -318,6 +330,8 @@ QString QgsPictureSourceLineEditBase::selectFileTitle() const
       return tr( "Select SVG File" );
     case Image:
       return tr( "Select Image File" );
+    case AnimatedImage:
+      return tr( "Select Movie File" );
```

Comment:
I think this might be misleading to refer to "movie" files here, I'd expect .mp4, .avi, etc. It'll also be a problem if-ever/whenever we add movie types to this widget in the future.

I suggest we settle on "Select Animated Image File".